src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,80 @@
+package com.google.sps.servlets;
+
+
+import java.lang.String;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+
+/**
+ * Responsible for collecting and storing contact-me.html form data
+ */
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Getting values entered in the form.
+    String userReview = Jsoup.clean(request.getParameter("review-input"), Whitelist.none());
+    boolean parking = false;
+    int noiseScore = 0;
+    int spaceScore = 0;
+    
+    //Checking which radio button is selected for noise level
+    switch(request.getParameter("noise").charAt(0)){
```

Comment:
This switch case and the next one look the same, I'd probably create a separate helper function.

private int getScore(char input) {...}